tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = cleanup, py{36,37,38}, style
+envlist = cleanup, py{36,37,38,39,py3}, style
```

Comment:
The py3 should be unnecessary, right? Let's remove it.

Reply:
Wouldn't that be needed for running tox locally with pypy? Or would that be included automatically?

Reply:
No need to test pypy3 locally by default. We can still use "tox -e pypy3" locally if we want to test it.

Reply:
Oh, okay. I'll remove that then.